whelktool/scripts/cleanups/2023/12/move-bearer-like-gfs-from-linked-work-to-instance.groovy

```diff
@@ -0,0 +1,97 @@
+/**
+ * LXL-4418
+ * Move bearer-like genre/form terms from work to instance like in
+ * ../../2020/08/lxl-3294-move-bearer-like-gfs-from-work-to-instance.groovy,
+ * however in this script we do it retroactively for works that have already been linked,
+ * i.e. the given genre/form term is removed from the linked work and instead put in all
+ * instances (of that linked work) that had the term in it's local work before.
+ */
+
+removedFromLinkedWork = getReportWriter("removed-from-linked-work.txt")
+addedToInstance = getReportWriter("added-to-instance.txt")
+
+gfsToMove = [
+        "E-böcker",
+        "E-single",
+        "E-textbok",
+        "Electronic books",
+        "Ljudbok cd",
+        "Ljudbok kassett",
+        "Ljudbok mp3",
+        "Musik cd",
+        "Tal cd",
+        "Talbok Daisy",
+        "Talbok kassett",
+        "Video dvd",
+        "Video vhs",
+        "Historiska faksimil",
+        "Deckare & spänning",
+        "Fantasy & SF",
+        "Idrott & friluftsaktiviteter",
+        "Hus & hem",
+        "Medicin & hälsa",
+        "Resor & geografi",
+        "Samhälle & politik"
```

Comment:
More like "uncontrolled" than "carrier-specific"; but we're currently moving those "out of sight" (not really logical (the content is still about or not about these), but that's another debate).

Reply:
True, "bearer-like" doesn't really apply no longer since those were added.